toc/rfc/rfc-draft-v2-app-manifests.md

```diff
@@ -0,0 +1,613 @@
+    service_offering:
+      name: my-service-offering
+    service_plan:
+      name: my-service-plan
+    tags:
+      - my-tag
+    parameters:
+      key1: value1
+    metadata:
+      annotations:
+        carrot: service instance metadata
+  - name: user-provided-service-instance
+    type: user-provided
+    credentials:
+      my-cred: secret1234
+    syslog_drain:
+      url: example.com/syslog
+    route_service:
+      url: example.com/route
+  - name: route-service-instance
+    service_offering:
+      name: route-service-offering
+    service_plan:
+      name: route-service-plan
+service_bindings:
+  - type: app
+    service_instance:
+      name: managed-service-instance
+    app:
+      name: app1
+    name: my-service-binding-name
+    parameters:
+      key1: value1
+      key2: value2
+    metadata:
+      annotations:
+        yam: service binding metadata
+  - type: app
+    service_instance:
+      name: managed-service-instance
+    app:
+      name: app2
+  - type: key
+    service_instance:
+      name: managed-service-instance
+    name: my-service-key
+  - type: route
+    service_instance:
+      name: route-service-instance
+    route:
+      url: route.example.com/path
+apps:
+  - name: app1
+    package:
+      path: /local/path/to/source/code
+      metadata:
+        annotations:
+          cassava: package metadata
+    lifecycle:
+      type: buildpack
+      buildpacks:
+        - name: ruby_buildpack
+        - name: java_buildpack
+        - url: git.example.com/my_buildpack
+      stack:
+        name: cflinuxfs4
+    environment_variables:
+      VAR1: value1
+      VAR2: value2
+    features:
+      ssh: true
+    metadata:
+      annotations:
+        contact: bob@example.com jane@example.com
+      labels:
+        sensitive: true
+    processes:
+      - type: web
+        command: start-web.sh
+        scale:
+          disk: 512M
+          instances: 3
+          memory: 500M
+        health_checks:
+          liveness:
+            http_endpoint: /health_check
+            type: http
+            invocation_timeout: 10
+            timeout: 10
+          readiness:
+            type: port
+        metadata:
+          annotations:
+            taro: process metadata
+      - type: worker
+        command: start-worker.sh
+        scale:
+          disk: 1G
+          instances: 2
+          memory: 256M
+        health_checks:
+          liveness:
+            type: process
+            timeout: 15
+  - name: app2
+    state: STOPPED
+    lifecycle:
+      type: docker
+    package:
+      docker:
+        image: docker-image-repository/docker-image-name
+        username: docker-user-name
+    processes:
+      - type: web
+        scale:
+          instances: 1
+          memory: 256M
+    sidecars:
+      - name: authenticator
+        processes:
+          - type: web
+          - type: worker
+        command: bundle exec run-authenticator
+        scale:
+          memory: 800M
+```
```

Comment:
Just to make sure https://github.com/cloudfoundry/community/pull/1019#issuecomment-2484034710 doesn't get lost. Please add a Cloud Native Buildpack example.

Reply:
Addressed by eb6a7a69dfc7bd4ca1fe2dee114e3171c767d492